splice_machine/src/main/java/com/splicemachine/derby/impl/stats/StoreCostControllerImpl.java

```diff
@@ -372,7 +395,8 @@ public static int getPartitions(byte[] table, List<Partition> partitions, boolea
             throw StandardException.plainWrapException(ioe);
         } finally {
             try {
-                root.close();
+                if (root != null)
+                    root.close();
```

Comment:
You can rewrite this with try-with-resources block

Reply:
Done.